lib/components/Column/Column.docs.tsx

```diff
@@ -14,38 +20,56 @@ const docs: ComponentDocs = {
         <Columns>
           <Column>
             <HideCode>
-              <Box
-                background="selection"
-                paddingTop="small"
-                paddingBottom="small"
-                paddingLeft="small"
-              >
-                <Text baseline={false}>Column</Text>
-              </Box>
+              <Content />
+            </HideCode>
+          </Column>
+          <Column>
+            <HideCode>
+              <Content />
+            </HideCode>
+          </Column>
+          <Column>
+            <HideCode>
+              <Content />
+            </HideCode>
+          </Column>
+        </Columns>
+      ),
+    },
+    {
+      label: 'Specific Widths',
+      render: () => (
+        <Columns>
+          <Column width="1/5">
+            <HideCode>
+              <Content>&#8533;</Content>
             </HideCode>
           </Column>
           <Column>
             <HideCode>
-              <Box
-                background="selection"
-                paddingTop="small"
-                paddingBottom="small"
-                paddingLeft="small"
-              >
-                <Text baseline={false}>Column</Text>
-              </Box>
+              <Content>Fluid</Content>
+            </HideCode>
+          </Column>
+          <Column width="1/3">
```

Comment:
We should probably make sure we have all widths covered for screenshot testing purposes, even if they're not all visible on the docs site.

Reply:
Done 👌